pages/device/[id]/device.js

```diff
@@ -17,6 +18,7 @@ const CATEGORY_MAP = {
   Lighting: <LightingIcon />,
 };
 export default function DetailsCard({ setDevices, devices, handleDelete }) {
+  const [showModalDelete, setShowModalDelete] = useState(false);
```

Comment:
Maybe `showDeleteModal` is nicer as `showModalDelete` is alomost the same as the name of your component?

Reply:
change it 😃